src/testUtils/mocks/dataSources.ts

```diff
@@ -672,6 +673,7 @@ function configureContext(opts: ContextOptions): void {
       },
     }),
     getSecondaryAccounts: sinon.stub().resolves(opts.secondaryAccounts),
+    getCurrentIdentity: sinon.stub().resolves(opts.getCurrentIdentity),
```

Comment:
This shouldn't be necessary (you added a `getCurrentIdentity` method to the current Identity instance returned by the context mock)

Reply:
If you want to override the result of `context.getCurrentIdentity`, you should use the `getCurrentIdentity` stub defined in this same function.

Something like:
```
if (opts.getCurrentIdentity) {
  // set the getCurrentIdentityStub to resolve to that value
} else {
  // define identity and use that
}
```

Keep in mind that that would override most of the ContextOptions